changelog.d/9536.bugfix

```diff
@@ -0,0 +1 @@
+Fix deleting pushers when using sharded pushers.
```

Comment:
is this the same as the newsfile for the PR that introduced the delta file? if not, please could it be?

Reply:
It is, sorry should have linked to it in the PR description: #9479